### PR TITLE
Fix shared dashboards

### LINKED ISSUE #5837: Shared dashboard URL ends up on an empty dashboard
## Bug description

Shared dashboard URL displays an empty dashboard even though the original dashboard has graphs.

## Expected behavior

See the same dashboard as the original

## How to reproduce

1. On a dashboard with some graphs click share
2. Open the URL to see an empty dashboard

Example original dashboard: https://app.posthog.com/dashboard/12795
Shared URL: https://app.posthog.com/shared_dashboard/bqEUx8VEhwFOexe9UyesIUTMcASUdw

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: _please provide_

### PR DESCRIPTION
**Pull request by tiina303**
## Changes

Closes https://github.com/PostHog/posthog/issues/5837
Fixes shared dashboard links, but I just duplicated the `setPageTitle` action from scenes.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by tiina303** [comment]
nvm this doesn't seem to make the page titles work

File: frontend/src/scenes/dashboard/dashboardLogic.tsx [hunk 1]
```diff
@@ -53,6 +52,9 @@ export const dashboardLogic = kea<dashboardLogicType>({
         setAutoRefresh: (enabled: boolean, interval: number) => ({ enabled, interval }),
         setRefreshStatus: (id: number, loading = false) => ({ id, loading }), // id represents dashboardItem id's
         setRefreshError: (id: number) => ({ id }),
+        setPageTitle: ({ title }) => {
+            document.title = title ? `${title} • PostHog` : 'PostHog'
```
Comment (paolodamico) [inline frontend/src/scenes/dashboard/dashboardLogic.tsx:56]: Almost there, it's just that actions in Kea need to be pure functions, so you need to actually pass the attribute name as we do with actions above and instead set up a `listener` with the same name as the action that actually executes this code.

Comment (tiina303) [reply frontend/src/scenes/dashboard/dashboardLogic.tsx]: 🤦‍♀️  thanks

**Review by paolodamico — APPROVED** [review]
Can't test here, but code lgtm! Thanks @tiina303

**Pull request merged**